### ISSUE TITLE
Error enabling app when using PostgreSQL

### ISSUE THREAD

**Issue by st33vil** [nextcloud/end_to_end_encryption#69]
When trying to enable app (version 1.0.5) on Nextcloud 13.0.1 with PostgreSQL 9.6.6 as a backend, I get the following error:
```
An exception occurred while executing 'CREATE UNIQUE INDEX id ON "oc_e2e_encryption_lock" ("id")':

SQLSTATE[42P07]: Duplicate table: 7 ERROR:  relation "id" already exists
```
The reason for this error is that the index cannot have the name ("id" in this case) already used as a column name. I solved the problem by changing the line 31 in the file /appinfo/database.xml so that the index name is "idx_id" instead of the "id". So the relevant part of the XML is as follows:
```
<index>
	<name>idx_id</name>
	<unique>true</unique>
	<field>
		<name>id</name>
		<sorting>ascending</sorting>
	</field>
</index>
```

**Comment by DanScharon** [comment]
this is still not fixed in Nextcloud 15

**Comment by st33vil** [comment]
Still not fixed in latest app version (1.3.0).
Simple rename of the index.name to "idx_id" from just "id" fixes the bug.

**Comment by georgehrke** [comment]
Pull-request in #129
